aten/src/ATen/native/native_functions.yaml

```diff
@@ -775,10 +775,6 @@
 
 - func: conv2d(Tensor input, Tensor weight, Tensor? bias=None, int[2] stride=1, int[2] padding=0, int[2] dilation=1, int groups=1) -> Tensor
 
-- func: _conv2d_prepack(Tensor weight, Tensor? bias=None, int[2] stride=1, int[2] padding=0, int[2] dilation=1, int groups=1, float? output_min=None, float? output_max=None) -> Tensor
-
-- func: _conv2d_packed(Tensor packed_weight, Tensor input) -> Tensor
```

Comment:
@gchanan @dzhulgakov @ezyang Are we OK with this?  I am no sure which way we decided to go eventually, but want to make sure we are not closing any doors for GPU in case there is benefit to pre-packing there as well (i.e. we have to do direct registration like this change is doing for GPU as well.)

Reply:
Ashkan, so we had the internal chat group discussion about this and my understanding is that we decided to go the way quantization is doing. To be fair, I did try to keep this change, however to make codegen work with custom class type will involve changes in quite a few places which become quite invasive. For custom class types I feel the current way is safer without exposing the custom type to broader source code. But if you guys have a different opinion, we can discuss.

Reply:
Not to spread FUD, but do you know if torchbind works on mobile?

Reply:
@ezyang after my stack of patches https://github.com/pytorch/pytorch/pull/34098 it should work fine on mobile